src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let money = 0;
+
+  if (days < 3) {
+    money = days * 40;
+  }
+
+  if (days > 2 && days < 7) {
+    money = days * 40 - 20;
+  }
+
+  if (days >= 7) {
+    money = days * 40 - 50;
```

Comment:
50 and 20 are mgic numbers, it's good to have explicit formulas to understand where these magic numbers are coming from.